app/github/utils.py

```diff
@@ -58,8 +58,8 @@ def build_auth_dict(oauth_token):
     """
     return {
         'api_url': settings.GITHUB_API_BASE_URL,
-        'client_id': settings.GITHUB_CLIENT_ID,
-        'client_secret': settings.GITHUB_CLIENT_SECRET,
+        'client_id': '90532f168702758eafe5',
```

Comment:
We intentionally don't hardcode values here.  You'll want to redact your keys throughout the PR.